docs/components/DocSlots.vue

```diff
@@ -53,13 +53,69 @@ export default {
         )} for detailed information.`
       },
       {
-        name: "before-list",
+        name: "control-icon",
         props: "-",
+        description: `Slot showed in the control start position. See ${link(
+          "#customize-control-icon"
+        )} for detailed information.`
+      },
+      {
+        name: "control-x",
+        props: makePropList(["xClass"]),
+        description: `Slot for custom control x template. See ${link(
+          "#customize-control-x"
+        )} for detailed information.`
+      },
+      {
+        name: "search-prompt-tip",
+        props: makePropList(["text"]),
+        description: `Slot for custom search prompt tip. See ${link(
+          "#customize-async-searching-tip"
+        )} for detailed information.`
+      },
+      {
+        name: "loading-tip",
+        props: makePropList(["text"]),
+        description: `Slot for custom loading tip`
+      },
+      {
+        name: "no-available-tip",
+        props: makePropList(["text"]),
+        description: `Slot for custom no available tip`
```

Comment:
Add "no available options", I was initially confused what "no available" meant :smile:

Reply:
Great suggestion! I also took the opportunity to update the “no results” as well.